pkg/controllers/internalmembercluster/v1beta1/member_controller.go

```diff
@@ -56,8 +56,8 @@ const (
 	// EventReasonInternalMemberClusterLeft is the event type and reason string when the agent left.
 	EventReasonInternalMemberClusterLeft = "InternalMemberClusterLeft"
 
-	// we add +-5% jitter
-	jitterPercent = 10
+	// we add +-7% jitter
+	jitterPercent = 14
```

Comment:
I don't get why increasing the jitter percent is going to help this case.

Reply:
https://github.com/Azure/fleet/actions/runs/7564476912/job/20598646789 the failure happened when we compared the requeueAfter. Relaxing the range should help. Adding error message should help us to figure out what's the diff.

Reply:
had a sync, this should not matter. We don't need to change this.